Codecademy/gamut
# feat(ColorMode): Adding "system" option to ColorMode

### Overview





### PR Checklist

- [X] Related to JIRA ticket: [WEB-1986](https://codecademy.atlassian.net/browse/WEB-1986)

#### Testing Instructions



1. Go to `{{URL}?path=/docs/foundations-colormode--page#system-color-mode` 
2. Verify new `system` color mode section accurately reflects the color mode currently active on your computer
3. Toggle the color mode in your computers system preferences and verify the changes are accurately reflected

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |
